webapp/crashstats/supersearch/views.py

```diff
@@ -284,20 +286,20 @@ def search_custom(request, default_context=None):
         except BadArgumentError as e:
             error = e
 
-    schema = settings.ELASTICSEARCH_INDEX_SCHEMA
-    now = timezone.now()
+    es_crashstorage = build_instance_from_settings(
+        socorro_settings.CRASH_DESTINATIONS["es"]
+    )
 
     possible_indices = []
-    for i in range(26):
```

Comment:
I reworked this code so it calls `get_indices` in the crash storage instance. That way things like the retention policy, how index names are generated, etc are all in the crash storage class and not spread across the code base.